meta/contributor-handbook.md

```diff
@@ -48,7 +48,8 @@ Patterns of level 2 are stored in the directory `/2-structured`.
 	- Patterns (`pattern-*.md`)
 	
 - Validation requirements:
-	- N/A
+	- Is partially validated by at least one known instance
+		- Or: pattern warrants to still be included (e.g. because validation is not feasible in the foreseeable future)

```

Comment:
If validation is not feasible, what outweighs that on the side of inclusion?
I would say that validation might be that some elements of the pattern have been validated even if the pattern as a whole hasn't been validated--i.e., there is some likelihood that this pattern will work if tried out. I cannot imagine a pattern where validation is simply not feasible in the foreseeable future.

Reply:
Good point. I wasn't able to come up with a pattern that's not feasible to validate after thinking for a bit. 
Maybe the phrasing here could be clarified or extended. 
Hinting to "Go, try it out."`*` was a good one here. Do you think you could create commit suggestion proposing a different/extended phrasing here @NewMexicoKid?

`*` One might want to add a few hints on how setup an try out such that it yields recordable or useful results wrt to the pattern, but that is something for a different document...

Reply:
How about:
- Validation requirements:
        - Is partially validated by at least one known instance
        - Has at least partial validation of individual solution elements when tried separately (i.e., there is objective reason to try the full solution) in the case where the full solution itself has not been validated

I think we don't have to hint at people trying out patterns; clearly, if they have a problem and see some benefit to trying a proposed solution, they will do so. We should, though, recommend that they update a pattern to note their experiences if they have tried it out.

Reply:
> If validation is not feasible, what outweighs that on the side of inclusion?
I would say that validation might be that some elements of the pattern have been validated even if the pattern as a whole hasn't been validated--i.e., there is some likelihood that this pattern will work if tried out. I cannot imagine a pattern where validation is simply not feasible in the foreseeable future.

I thought about this question quite long and I think @NewMexicoKid you are right. Our definition of validation is that of the pattern being used successfully. That in itself is a pragmatic way of validation.

If validation is not feasible, then this means we believe the practice cannot be implemented and, in consequence, we should not let it reach a higher maturity level.

Reply:
@NewMexicoKid, @lenucksi : I used Tim's formulation as the base and adapted the text. Please resolve if that works for you :)